spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/person'
+
+describe Person do
+  books = [{ title: 'Managment101', author: 'Rich' }, { title: 'Cooking101', author: 'Fat'}]
+
+  it 'constract of empty array at start' do
+    expect(subject.portfolio).to eq []
+  end
+
+#  context 'adding book to portfolio' do
+#    before { allow(library).to receive(:check_out).and_return({ }) }
+
+#    it 'searched by :title' do
+#      subject.rent(library, title: 'Cooking101')
+#      expect(subject.portfolio).to eq [{ title: 'Cooking101', author: 'Fat' }]
+#    end
+
+#    it 'searched by :author' do
+#      subject.rent(library, author: 'Rich')
+#      expect(subject.portfolio).to eq [{ title: 'Managment101', author: 'Rich' }]
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)